CourseManager/Courses/Course 4 UI and Animations in SwiftUI/Utils/ExtensionsWeather.swift

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
It is better to name file Weather + Extensions. Additionally, if you use colors only in weather, it is better to either add computed property/function directly to weather object

Reply:
Hi Yura, I've updated the name of the file. Just want to double check I understood the second part correctly. I'm using the colors across the weather app and not changing any colors based on changes in the weather or forecast objects. Should I still be putting the colors as computed properties/functions in the weather object?

Reply:
Hi, it is better, that the weather will have `state` variable and which will provide color. For example, `state` is enum and it has associated color